docs/source/index.rst

```diff
@@ -1,83 +1,103 @@
-HuggingFace Datasets
-=======================================
+Datasets
+========
 
-Datasets and evaluation metrics for natural language processing
+.. image:: /imgs/datasets_logo.png
+   :align: center
 
-Compatible with NumPy, Pandas, PyTorch and TensorFlow
+🤗 Datasets is a library for easily accessing and sharing datasets, and evaluation metrics for Natural Language Processing (NLP). 
```

Comment:
General question for @lhoestq @albertvillanova : do we want `datasets` to be known just for NLP or should we expand the scope? For example we could write:

> ...  for Natural Language Processing (NLP) and beyond.